Updating Getting Certificate files

Reni-90 opened the pull request:
### Change Description ###

As part of updating outdated contents for Tizen .NET files, updated the following files:
- getting-the-certificates.md
- installing-certificate-extension.md
- permit-device-to-install-apps.md
- certification_guide2.png

Please review the files for technical accuracy.

Some doubts and suggestions are given as comments. Please verify them and give you valuable feedback.

Reni-90 commented on docs/application/dotnet/tutorials/certificates/permit-device-to-install-apps.md:
```diff
@@ -1,15 +1,16 @@
+3. Select **Permit to install applications** from the options.
 
-![Device Manager](media/DeviceManager.png)
+    ![Device Manager](media/DeviceManager.png)
```
Request you to provide an updated screenshot capturing the latest version.

hakkyu-samsung replied:
We believe there is no major difference, but we will include the latest image.

We cannot push commits to a public repository inside the company network, instead we will provide a url for the image.

https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/permit-device-to-install-apps-2.png

Reni-90 replied:
Thank you.

Reni-90 commented on docs/application/dotnet/tutorials/certificates/permit-device-to-install-apps.md:
```diff
@@ -1,15 +1,16 @@
 # Permit Device To Install Applications
 
```
Suggestion:
Request you to provide an introductory line for this section.

hakkyu-samsung replied:
We realized that this article "permit-device-to-install-apps.md" is actually for older Galaxy Watch series that don't support .NET platform. We should remove it from the documentation.

Reni-90 replied:
@hakkyu-samsung
Thank you for the explanation.
I will do as suggested.

Reni-90 commented:
@mijin85cho 
Thank you for notifying me about [#1112](https://github.com/Samsung/tizen-docs/issues/1112).
The first part mentioned: "This is the current picture in Installing Certificate Extension but I think it should highlight Samsung Certificate Extension menu."
The image is updated in this PR.

For the second part, as you suggested, I will wait for @hakkyu-samsung's comment. Based on the comment, I will update the PR.

hakkyu-samsung commented:
Hello, @Reni-90 and @mijin85cho.
Thank you for the suggestions and the updates!

I'm currently looking in the the second part of #1112, I'll write a guide and add it to the comments within today.

hakkyu-samsung commented:
I've wrote the explanation for the second part of #1112.
I'm currently not sure where to add the new contents. Since app signing will be the next step after creating the certificates, how about posting it on a new page just after "Managing Certificate Profile"?
The filename and contents are written below:

signing-your-application-with-a-certificate.md

# Signing your Application with a Certificate

After creating samsung certificate for your galaxy watch, you need to make sure that Visual Studio is using that certificate to sign your application. , you need to make sure that Visual Studio is using that certificate to sign your application. 

Take the following steps to check your cofiguration:

1. In Visual Studio, click **Tools > Options**.

   ![](https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/installing-certificate-extension-4.png)

2. In the left sidebar, locate **Tizen > Certificate**.

3. Check the checkbox: **"Sign the .TPK file using the following option"**.

4. Select **"Use profile of Tizen Certificate Manager"** from the drop-down menu below the checkbox.

5. In the **Profile** section, set the certificate profile that you created, and click **OK**.

   ![](https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/installing-certificate-extension-3.png)
----

Reni-90 commented:
@hakkyu-samsung 
Thank you for the suggestions and content for the new file.
I have updated the PR based on your suggestions:
1. Removed the permit-device-to-install-apps.md file based on https://github.com/Samsung/tizen-docs/pull/1108#discussion_r429063945 and updated the getting-the-certificates.md file.
2. Added a new file, signing-your-application-with-a-certificate.md based on https://github.com/Samsung/tizen-docs/pull/1108#issuecomment-633426037.
3. Updated the getting-the-certificates.md file based on https://github.com/Samsung/tizen-docs/pull/1108#issuecomment-633426037. Changed the heading to align with other files.

Please verify the changes.

Reni-90 commented:
@hakkyu-samsung 
Thank you.
I will update.

Reni-90 commented on docs/application/dotnet/tutorials/certificates/getting-the-certificates.md:
```diff
@@ -19,17 +15,19 @@ Contents:
+
+> [!NOTE]
+> If you upload your applications to the respective stores without the original author certificate that signs your app, you will not be able to publish any updates on your applications.
+
```
@mijin85cho and @hakkyu-samsung 
Thank you for the clarification.

Reni-90 commented:
@hakkyu-samsung,
As discussed, I have updated changes.
Please verify.

mijin85cho commented on docs/application/dotnet/tutorials/certificates/signing-your-application.md:
```diff
@@ -0,0 +1,15 @@
+4. Select **Use profile of Tizen Certificate Manager** from the drop-down list as highlighted in the image.
+5. In the **Profile** section, select the certificate profile that you have created, and click **OK**.
+	
+    ![Options](media/certification_guide27.png)
```
I can see the image is captured from the Windows in Korean.

![image](https://user-images.githubusercontent.com/9958374/84098298-dddbff00-aa41-11ea-829a-8bd10ef47287.png)

As we are guiding to capture the images from the Windows in Eng, 
@hakkyu-samsung , would you please capture this image again from the Windows in Eng if possible?
In the Windows in Eng, the file path is marked with \\, not ￦ as in Windows in Korean.

hakkyu-samsung replied:
@mijin85cho 
Thanks for the suggestion.
This may take some time, since I have to set up Windows in Eng, and the dev environment.
I will upload it during this week.

Reni-90 replied:
@hakkyu-samsung
Thank you.
I will update the changes with the new image.

hakkyu-samsung replied:
@Reni-90 

Providing the new images:
* https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/certification-guide26.png
* https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/certification-guide27.png

Reni-90 replied:
@hakkyu-samsung 
Thank you, I will update the images and the changes.

mijin85cho commented on docs/application/dotnet/tutorials/certificates/getting-the-certificates.md:
```diff
@@ -1,35 +1,34 @@
 - [Installing Certificates Extension](installing-certificate-extension.md)
 - [Creating Certificates](creating-certificates.md)
-- [Permit Device To Install Applications](permit-device-to-install-apps.md)
+- [Signing Application with Certificate](signing-your-application-with-a-certificate.md)
```
Would you please check this link again?
I think the file name is changed to signing-your-application.md.

Reni-90 replied:
Thank you for pointing out.
I will fix the link.

Reni-90 commented on docs/application/toc_all.md:
```diff
@@ -29,7 +29,7 @@
 #### [Installing Certificates Extension](/application/dotnet/tutorials/certificates/installing-certificate-extension.md)
 #### [Creating Certificates](/application/dotnet/tutorials/certificates/creating-certificates.md)
-#### [Permit Device To Install Applications](/application/dotnet/tutorials/certificates/permit-device-to-install-apps.md)
+#### [Signing Application with Certificate](/application/dotnet/tutorials/certificates/signing-your-application.md)
```
Yes, we can.

Reni-90 commented on docs/application/dotnet/tutorials/certificates/permit-device-to-install-apps.md:
```diff
@@ -1,15 +0,0 @@
-
-Right-click anywhere in the file explorer of the **Device Manager** and select **Permit to install applications**.
-
-![Device Manager](media/DeviceManager.png)
```
Sure, I will remove this image.

Reni-90 commented:
@hakkyu-samsung and @mijin85cho 
I have updated the changes.
Please verify.

mijin85cho commented:
Thanks for the updates and the review. However I have one question.

> **hakkyu-samsung**
...
Since app signing will be the next step after creating the certificates, how about posting it on a new page just after "Managing Certificate Profile"?


For now, the "signing application" page is located between "creating certificates" and "managing certificate profile" unlike @hakkyu-samsung's suggestion. 
@hakkyu-samsung would you please confirm this placement? Actually I agree with your previous suggestion that placing "signing application" next to "managing certificate profile" is more natural, but current placement is also acceptable anyway.

**getting-the-certificates.md :**
![image](https://user-images.githubusercontent.com/9958374/84970283-4e2b0480-b155-11ea-8ce9-a0c534ab6a75.png)

**toc_all.md :**
![image](https://user-images.githubusercontent.com/9958374/84970315-613dd480-b155-11ea-80e2-9dbb151e5676.png)

hakkyu-samsung commented:
@mijin85cho 
Thanks for pointing this out. I actually forgot that I had suggested the placement.

I think the current placement looks better though I have originally suggested a different placement. From a developer's perspective, managing the certificate profile feels optional or something that can be done later. On the other hand, "signing application" is the direct purpose of making the certificate, so it should be placed closer to the "creating certificates" page. This will allow the readers to achieve their goal quickly as possible by following the guide in a sequence.

If you have other suggestions, plz share your thoughts:)

Reni-90 commented:
@hakkyu-samsung and @mijin85cho 
Thank you very much for the review and suggestions.
